config/config.proto

```diff
@@ -25,6 +25,7 @@ import "gogoproto/gogo.proto";
 option (gogoproto.sizer_all) = true;
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unrecognized_all) = false;
```

Comment:
Can you add some justification to the commit message about why we want to remove unrecognized value support?